tests/end2end/cypress/integration/cmdline-wmts-ghaction.js

```diff
@@ -170,205 +170,124 @@ describe('WMTS command line', function () {
-            .should('not.contain', 'Missing parameter "project"')
-            .should('not.contain', 'Missing parameter "layers"')
-            .should('not.contain', 'Missing parameter "TileMatrixSet"')
-            .should('not.contain', 'Missing parameter "TileMatrixMin"')
-            .should('not.contain', 'Missing parameter "TileMatrixMax"')
-            .should('not.contain', 'The QGIS project /srv/lzm/tests/qgis-projects/tests/unknown.qgs does not exist!')
-
-        cy.exec('./../lizmap-ctl docker-exec truncate -s 0 /srv/lzm/lizmap/var/log/errors.log')
-        cy.exec('./../lizmap-ctl script lizmap~wmts:seeding -v -f -dry-run testsrepository unknown Quartiers EPSG:3857 10 10', {failOnNonZeroExit: false})
+        cy.exec('./../lizmap-ctl console wmts:cache:seed -v -f --dry-run testsrepository unknown Quartiers EPSG:3857 10 10', {failOnNonZeroExit: false})
             .then((result) => {
                 expect(result.code).to.equal(1)
                 expect(result.stdout).to.contain('The project has not be found!')
             })
 
-        cy.exec('./../lizmap-ctl docker-exec cat /srv/lzm/lizmap/var/log/errors.log')
-            .its('stdout')
-            .should('not.contain', 'Missing parameter "repository"')
-            .should('not.contain', 'Missing parameter "project"')
-            .should('not.contain', 'Missing parameter "layers"')
-            .should('not.contain', 'Missing parameter "TileMatrixSet"')
-            .should('not.contain', 'Missing parameter "TileMatrixMin"')
-            .should('not.contain', 'Missing parameter "TileMatrixMax"')
-            .should('not.contain', 'The QGIS project /srv/lzm/tests/qgis-projects/tests/unknown.qgs does not exist!')
-
+        
         // Clear errors
         cy.exec('./../lizmap-ctl docker-exec truncate -s 0 /srv/lzm/lizmap/var/log/errors.log')
 
-        cy.exec('./../lizmap-ctl script lizmap~wmts:seeding -v -f -dry-run testsrepository cache unknown EPSG:3857 10 10', {failOnNonZeroExit: false})
+        cy.exec('./../lizmap-ctl console wmts:cache:seed -v -f --dry-run testsrepository cache unknown EPSG:3857 10 10', {failOnNonZeroExit: false})
             .then((result) => {
                 expect(result.code).to.equal(1)
                 expect(result.stdout).to.contain('The layers \'unknown\' have not be found!')
             })
 
         cy.exec('./../lizmap-ctl docker-exec truncate -s 0 /srv/lzm/lizmap/var/log/errors.log')
-        cy.exec('./../lizmap-ctl script lizmap~wmts:seeding -v -f -dry-run testsrepository cache Quartiers unknown 10 10', {failOnNonZeroExit: false})
+        cy.exec('./../lizmap-ctl console wmts:cache:seed -v -f --dry-run testsrepository cache Quartiers unknown 10 10', {failOnNonZeroExit: false})
             .then((result) => {
                 expect(result.code).to.equal(1)
                 expect(result.stdout).to.contain("The TileMatrixSet 'EPSG:3857'!\nThe TileMatrixSet 'unknown' has not be found!")
             })
 
```

Comment:
Can you add a test about `TileMatrixMin and TileMatrixMax must be of type int`
```suggestion

        cy.exec('./../lizmap-ctl console wmts:cache:seed -v -f --dry-run testsrepository cache Quartiers EPSG:3857 TileMatrixMin 10', {failOnNonZeroExit: false})
            .then((result) => {
                expect(result.code).to.equal(1)
                expect(result.stdout).to.contain("TileMatrixMin and TileMatrixMax must be of type int")
            })

        cy.exec('./../lizmap-ctl console wmts:cache:seed -v -f --dry-run testsrepository cache Quartiers EPSG:3857 10 TileMatrixMax', {failOnNonZeroExit: false})
            .then((result) => {
                expect(result.code).to.equal(1)
                expect(result.stdout).to.contain("TileMatrixMin and TileMatrixMax must be of type int")
            })
            
```